### DISCUSSION TITLE
Expiring claims announcement into game chat

### DISCUSSION THREAD

**Discussion by Zoneman1972** [GriefPrevention/GriefPrevention#474] [Ideas]
Hey there,
I was wondering if there is a chance that you could add in the ability to have a message broadcast into the game chat notifying when a players claim has expired. http://prntscr.com/mh7z8a
I see the message in the console and I was hoping that this could be adjusted for the game chat but there is no current setting for it.
The main reason is to notify players that a new raidable base has come up for grabs and they can all scramble and fight for the goods.
Thanks for your time
Zonemann

**Comment by 112RG** [comment]
I can make this into an addon if you would like to shoot me a PM on Discord 112#3560
